ansible/roles/test/files/ptftests/py3/hash_test.py

```diff
@@ -41,6 +41,7 @@ class HashTest(BaseTest):
     # Class variables
     # ---------------------------------------------------------------------
     DEFAULT_BALANCING_RANGE = 0.25
+    RELAXED_BALANCING_RANGE = 0.80
```

Comment:
@arista-nwolfe - how did we arrive at this number?

Reply:
@tjchadaga I went through our results from the past 3 months and we were never higher than `0.80` on our T2 devices for `ip-proto`.